website/docs/components/pagination/partials/guidelines/guidelines.md

```diff
@@ -18,17 +18,54 @@ Cursor-based pagination allows users to navigate to the next or previous set of
 Offset or page-based pagination divides a dataset into pages containing a default or user-determined number of records, and allows users navigate to any particular page. In most cases, the numbered pagination provides a better user experience. It allows users to jump between pages and always return to the first page or go to the last page without navigating through the pages manually.
 
 ![Numbered pagination example](/assets/components/pagination/pagination-offset-example.png)
-- Supported by offset (page-based) pagination.
+
+_Supported by offset (page-based) pagination._
 
 ![Compact pagination example](/assets/components/pagination/pagination-cursor-example.png)
-- Supported by offset and cursor based pagination.
+
+_Supported by offset and cursor based pagination_
 
 !!! Warning 
 
 We strongly suggest that you talk to your engineering team to see which pagination variant is right for your project.
 
 !!!
 
+### Truncation
+
+By default, in Numbered Pagination, the number of visible pages will be truncated when the total number of pages exceeds seven. What pages are truncated depends on the current page the user is on, with a few notable constants:
+
+- The first and last page will always be displayed (never be truncated).
+- The previous page and next page compared to the current page will always be displayed (unless the current page is the first or last page).
+- A maximum of seven pages or truncated pages will always be displayed.
+
+!!! Info
+
+The number of pages equals `total items / items per page`. E.g., if the total number of items is 120 and there are 10 items per page, the number of pages is 12. However, this can be variable depending on the `PageSize` and can determine whether the number pages extends beyond the threshold of truncation.
```

Comment:
```suggestion
The number of pages equals `total items / items per page`, e.g., if the total number of items is 120 and there are 10 items per page, the number of pages is 12. However, this can be variable depending on the `PageSize` and can determine whether the number of pages extends beyond the threshold of truncation.
```

Reply:
Ah yeah, good call. I did a small amount of cleanup of the non-new content, I'll adjust the hierarchy as well but I'll note that having an `h3` above each one of the truncation images is not great, so I'll clean up that content a bit too.